end-to-end-tests/cypress/e2e/OSCAL/Drawer.cy.js

```diff
@@ -48,39 +61,35 @@ describe("The Viewer", () => {
   });
 
   it("loads a catalog", () => {
```

Comment:
```suggestion

  it("loads a catalog in REST mode", () => {
```

A different test already checks whether a catalog is loaded, so it seems appropriate to be more specific here (and other places in this PR) to rename it to loading in REST mode.

Reply:
I realize this isn't initially part of the PR itself, just something I wanted to point out.

Reply:
So the `describe` deals with that. It reads `The Editor loads a catalog` & `The Viewer loads a catalog`. Being in or out of REST mode is not super descriptive of what is actually happening (using the Viewer or Editor).